pubpub/pubpub
# feat: add ui for analytics selection

## Issue(s) Resolved

#2995 

## Test Plan

1. Go to `jtrialerror` & open network tab
2. Go to `/dash/settings/analytics-settings`
3. Change to `default`
4. Hit save
5. Go to home
6. Confirm no events are sent to `googletagmanager.com`
7. Go back to `/dash/settings/analytics-settings`
8. Change to "Simple Analytics"
9. Hit save
10. Go Home
11. ...
12. go back
13. Change back to "Google Analytics" with Measurement ID `G-9GK39XDD27`
14. Hit save
15. Test connection
16. Go to home
17. Confirm events are sent to google analytics


### Non-enrolled communities should not see this setting

1. go to `demo`
2. Go to `/dash/settings/details`
3. Observe that no `Analytics` tab is present
4. Try to manually go to `/dash/settings/analytics-settings`
5. Get redirected to `/dash/settings/details`


## Screenshots (if applicable)

## Optional

### Notes/Context/Gotchas

### Supporting Docs
